src/AnnotatedCommand/CommandInfoAlterer.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
This isn't how I would inject ignored method names into Annotated Command. Instead, I'd make a field of ignored names or ignored regex's in the AnnotatedCommandFactory, add a setter / appender for that, and call it in the DI constructor.

Since the currentState and restoreState methods are part of a Robo trait, though, you can just annotate them with @ignored-command and remove this.

If you don't want to use the annotation, then please make a PR in annotated-commands to add an ignored name / ignored regex facility to AnnotatedCommandFactory.

Reply:
Thank you for the input.
I'll bring this conversation to the annotated-command repo with a PR, just a few notes here so I write better code.

I will go with the ignored regex facility but for that I also need to convert to non-static the `isCommandMethod` in the factory.
I am not familiar with configuring the container but I'll follow the examples I see around these lines:
https://github.com/consolidation/Robo/blob/3.x/src/Robo.php#L324

Am I correct?